charts/incubator/bar-assistant/values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  repository: barassistant/server:v3
+  pullPolicy: IfNotPresent
+  tag: 
```

Comment:
Missing tag
```suggestion
  tag: v3@sha256:63c9309aafdf38d6bc4a1049d2655f4db6a7d55eb0a6059daaa25e98adb4b10f
```

Reply:
Those can be found on the dockerhub page for this container: https://hub.docker.com/r/barassistant/server/tags